README.md

```diff
@@ -250,7 +250,7 @@ docker login ${REGISTRY} -u ${UPBOUND_ACCOUNT_EMAIL}
 Build package.
 
 ```console
-up xpkg build --name package.xpkg --ignore "examples/*,hack/*"
+up xpkg build --name package.xpkg --ignore ".github/*,.github/*/*,examples/*,hack/*"
```

Comment:
This is one of my least favorite parts of `filepath.Match` (ref: https://github.com/golang/go/issues/11862) ☹️ Might consider extending to make recursive ignore supported in our CLI

Reply:
Yeah, it was confusing, took me a while to figure out.